go.mod

```diff
@@ -7,12 +7,15 @@ require (
 	github.com/matryer/moq v0.3.1
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.8.0
+	github.com/tendermint/tendermint v0.34.2
```

Comment:
should be `v0.34.21`

Reply:
are there any issues with using this version? it's literally just used to include the logger interface

Reply:
It adds more clashing versions in go.mod. not sure if it breaks anything in this case, but I have kept dep version consistent for the important deps like tendermint/cosmos sdk